include/genn/genn/code_generator/codeGenUtils.h

```diff
@@ -80,6 +81,12 @@ GENN_EXPORT std::string ensureFtype(const std::string &oldcode, const std::strin
 //--------------------------------------------------------------------------
 GENN_EXPORT void checkUnreplacedVariables(const std::string &code, const std::string &codeName);
 
+//--------------------------------------------------------------------------
+/*! \brief This function substitutes function names in a code with namespace as prefix of the function name for backends that do not support namespaces by checking that the function indeed exists in the support code and returns the substituted code.
+ */
+ //--------------------------------------------------------------------------
+GENN_EXPORT std::string substituteNamespaceFunction(const std::string supportCode, const std::string code, std::string namespaceName);
+
```

Comment:
For consistency with other functions like this in GeNN, I think:

```c++
GENN_EXPORT void disambiguateNamespaceFunction(std::string &supportCode, const std::string &namespaceName);
```

might be a better signature and name.